packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenuItemGroup.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+export interface OptionsMenuItemGroupProps extends React.HTMLProps<HTMLLIElement> {
```

Comment:
Extending HTMLLIElement then I believe your `<li>` needs to add `{...props}`

Reply:
Fixed!